.env

```diff
@@ -1,5 +1,5 @@
 PORT=3000
 ETHCONNECT_URL=http://127.0.0.1:5102
+ETHCONNECT_INSTANCE=/contracts/erc1155
```

Comment:
Wow, this one is old. Why did it pop back up? Do we still rely on it anywhere? Definitely tried to move to `CONTRACT_ADDRESS` a while back.